### PR TITLE
fix: update the table header to sticky [#56]

### PR DESCRIPTION
**Pull request by meshkat-hossain**
Resolved : #56 
- Here i am making the code editor test result viewer table header sticky.

sticky

### CODE REVIEW

**Comment by mimukit** [comment]
Quick tips: for PR title:

- Just start the title with verb.

Example:

```diff
- fix: Here i am making the table header sticky [#56]

+ fix: update the table header to sticky [#56]
```

File: src/components/CodeEditor/TestResultViewer.tsx [hunk 1]
```diff
@@ -57,13 +57,16 @@ const TestResultTable = ({ editorOutput }) => {
       </div>
       <div className="h-48 overflow-auto">
         <table className="m-0 min-w-full table-auto border dark:border-slate-800/60">
-          <thead className="bg-gray-100 dark:bg-slate-800/60">
-            <tr>
+          <thead className="bg-white sticky top-0 dark:bg-slate-800/60">
```
Comment (mimukit) [inline src/components/CodeEditor/TestResultViewer.tsx:60]: Previously the background color was `bg-gray-100`; why did we change the color to `bg-white`? Is there any specific reason?

File: src/components/CodeEditor/TestResultViewer.tsx [hunk 2]
```diff
@@ -57,13 +57,16 @@ const TestResultTable = ({ editorOutput }) => {
...
+            <tr >
               {resultTableColumns.map((column) => (
                 <th
                   key={column.key}
                   scope="col"
-                  className="border border-gray-200 p-2 text-center text-xs text-slate-600 dark:border-slate-600/80 dark:text-slate-400"
+                  className=" border-gray-200 p-2 text-center text-xs text-slate-600 dark:border-slate-600/80 dark:text-slate-400 after:content-[''] after:absolute  after:top-0 after:left-0  after:w-full after:h-8 after:border after:border-gray-200
```
Comment (mimukit) [inline src/components/CodeEditor/TestResultViewer.tsx:66]: The formatting of all these class names needs to be corrected. The ordering of those class names needs to be appropriately sorted. We use the tailwind-prettier plugin to do the sorting automatically in the codebase. Your vscode is not probably setup properly to work with that.

Please have a look into this doc to setup the tailwind-prettier plugin in your machine: https://tailwindcss.com/blog/automatic-class-sorting-with-prettier

**Review by mimukit — CHANGES_REQUESTED** [review]
1. The style needs to be fixed on the dark mode theme. After scrolling, the text behind the header is visible and overlapping.

CleanShot 2023-02-21 at 14 23 47

2. On the bright theme, there is a border thickness mismatch. This needs to be fixed.

CleanShot 2023-02-21 at 14 20 22

**Comment by mimukit** [comment]
Also before creating PR, please make sure your feature branch is up-to-date with `main` branch.

File: src/components/CodeEditor/TestResultViewer.tsx (hunk 2 above)
Comment (meshkat-hossain) [reply src/components/CodeEditor/TestResultViewer.tsx:66]: Thanks for your observation, tailwind prettier setup done.

File: src/components/CodeEditor/TestResultViewer.tsx (hunk 1 above)
Comment (meshkat-hossain) [reply src/components/CodeEditor/TestResultViewer.tsx:60]: Sorry for that, its my mistake. i already changed this.

**Comment by meshkat-hossain** [comment]
resolved #62 
-fixing the table header border thickness.
-fixing the overlapping problem in dark mood.
Screenshot (4)
Screenshot (5)

**Review by mimukit — APPROVED** [review]

**Pull request merged**